client/src/components/core/ms-dropdown/MsDropdown.vue

```diff
@@ -22,7 +22,7 @@
         slot="end"
         :icon="getIcon()"
       />
-      <span class="input-text">{{ labelRef }}</span>
+      <span class="input-text">{{ $t(labelRef || '') }}</span>
```

Comment:
Empty keys create warnings
`[intlify] Not found '' key in 'fr-FR' locale messages.`